citydb-config/src/main/java/org/citydb/config/ConfigManager.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * citydb-tool - Command-line tool for the 3D City Database
+ * https://www.3dcitydb.org/
+ *
+ * Copyright 2022-2023
+ * virtualcitysystems GmbH, Germany
+ * https://vc.systems/
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.citydb.config;
+
+import com.alibaba.fastjson2.JSON;
+import com.alibaba.fastjson2.JSONException;
+import com.alibaba.fastjson2.JSONObject;
+
+import java.io.BufferedInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Objects;
+
+public class ConfigManager {
+    private static final ConfigManager instance = new ConfigManager();
+
+    private Config config;
+
+    private ConfigManager() {
+    }
+
+    public static ConfigManager getInstance() {
+        return instance;
+    }
+
+    public Config getConfig() {
+        return config;
+    }
+
+    public void load(Path path) throws ConfigException {
```

Comment:
shouldn't the ConfigManager return a Config object here rather than holding a reference to a single Config object? The method "getConfig" is not required then anymore.